content/sublease.md

```diff
@@ -0,0 +1,52 @@
+# SUBLEASE AGREEMENT
+
+This Sublease Agreement (the "Agreement") is made on [DATE] between G. Ochoa LLC ("Sublessor") and [SUBLESSEE NAME] ("Sublessee").
+
+WHEREAS, Sublessor leases certain premises located at 544 Park Ave, Suite 332, New York, NY 12205 (the "Premises") and wishes to sublease a portion of the Premises to Sublessee.
+
+NOW, THEREFORE, in consideration of the mutual covenants herein, the parties agree as follows:
+
+1. SUBLEASED SPACE: Sublessor agrees to sublease to Sublessee a portion of the Premises (the "Subleased Space") for activities.
+
+2. TERM: The term of this Sublease shall commence on July 1st, 2024 and end on May 30th, 2025, unless terminated earlier as provided herein.
+
+3. RENT: Sublessee shall pay the amount stipulated by the Brooklyn Industrial Arts Treasurer every month, due on the first day of each month.
```

Comment:
Add a note to explain how this is calculated, i.e. that it is the total rent plus approved expenses divided by the number of members

Reply:
I actually don't think we want that in writing because then we can't make changes to it without reissue leases. I think we should leave it out.